controller/feature.go

```diff
@@ -1,49 +1,65 @@
 package controller
 
 import (
+	jwtgo "github.com/dgrijalva/jwt-go"
 	"github.com/fabric8-services/fabric8-toggles-service/app"
-	"github.com/fabric8-services/fabric8-toggles-service/configuration"
 	"github.com/fabric8-services/fabric8-toggles-service/errorhandler"
+	"github.com/fabric8-services/fabric8-toggles-service/errors"
+	"github.com/fabric8-services/fabric8-toggles-service/featuretoggles"
+	"github.com/fabric8-services/fabric8-wit/log"
 	"github.com/goadesign/goa"
-	uuid "github.com/satori/go.uuid"
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
```

Comment:
are you sure you need both `jwt` libraries ?

Reply:
similar to `features.go` we need `goajwt` to extract the token from middleware, the other one is to decode JWT token

Reply:
ok, then